src/features/DevTools/slice.ts

```diff
@@ -0,0 +1,40 @@
+import { createAction, createReducer } from '@reduxjs/toolkit';
+import { call, takeEvery } from 'redux-saga/effects';
+
+import { path } from '@vendor';
+
+const sliceName = 'demo';
+const initialState = {
+  greeting: 'Hello from Reducer',
+  count: 0
+};
+
+/* Selectors */
+export const getGreeting = path([sliceName, 'greeting']);
+export const getCount = path([sliceName, 'count']);
+
+/* Actions */
+export const increment = createAction<number | undefined>(`${sliceName}/increment`);
+export const reset = createAction<undefined>(`${sliceName}/reset`);
+
+/* Sagas */
+/** Saga - worker */
+function* onIncrement() {
+  yield call(console.log, 'Hello from Saga!');
```

Comment:
Just an fyi, there's no benefit in calling `console.log` using the `call` effect over simply writing `console.log('Hello from Saga!');`

Reply:
Yes, It's just a reminder to use `call` for testing side-effects.